mlflow/autogen/__init__.py

```diff
@@ -108,7 +108,63 @@ async def patched_agent(original, self, *args, **kwargs):
 
 def _convert_value_to_dict(value):
     # BaseChatMessage does not contain content and type attributes
-    return value.model_dump(serialize_as_any=True) if isinstance(value, BaseModel) else value
+    if isinstance(value, BaseModel):
```

Comment:
No, we should log all the available attributes and should not change the actual tracing logic. Can you fix the test by having a list of available attribute keys for different versions, or by not comparing new attributes during the assertion?

Reply:
@B-Step62 can you repeat this?